Documentation/Contributing.md

```diff
@@ -203,5 +252,55 @@ path which exists on disk. If none of the paths are present on disk, it will fal
 back to built-in computation.
 
 ## Making changes in TSC targets
-
 All targets with the prefix TSC define the interface for the tools support core. Those APIs might be used in other projects as well and need to be updated in this repository by copying their sources directories to the TSC repository. The repository can be found [here](https://github.com/apple/swift-tools-support-core).
+
+## Commits / PRs
```

Comment:
rename section "Creating Pull Requests" and move after "Testing on Linux with Docker"